rasa_sdk/forms.py

```diff
@@ -49,134 +52,17 @@ def required_slots(tracker: "Tracker") -> List[Text]:
             "A form must implement required slots that it has to fill"
```

Comment:
Can't we deprecate the entire `FormAction` as it was only used with the old `FormPolicy` which was dropped with Rasa Open Source 3.0?

Reply:
+1, considering the warning message in this `FormAction` constructor, this was deprecated in 2.0 so it can be safely removed now, alongside tests? Also, then the `SlotMapping` code can also be removed.